pkg/profile/utils.go

```diff
@@ -153,9 +153,7 @@ func StringListFromCommaSeparated(cs string) (sl []string) {
 	if cs == "" {
 		return
 	}
-	for _, item := range strings.Split(cs, ",") {
-		sl = append(sl, item)
-	}
+	sl = append(sl, strings.Split(cs, ",")...)
```

Comment:
It will be good to have a test for this.

Reply:
The reason for not doing it is because this is used by 0.3 and 0.9 features. Neither app nor app-profiles rely on it, and we have an issue to remove those features.